### PR TITLE
Rename management API to Umbraco.Cms.Api.Management

### PR DESCRIPTION
**Pull request by kjac**
### Description

Rename management API project from `Umbraco.Cms.ManagementApi` to `Umbraco.Cms.Api.Management` and update namespaces etc. accordingly.

### CODE REVIEW

**Review by elit0451 — APPROVED** [review]
The changes look good and my tests didn't yield any strange behaviour 💪 . I've added a few comments regarding renaming `ManagementApi*` to `ApiManagement*` 🙂 

Apart from my comments on the changed files, there is one more case to consider if we want to exchange the places of **management** and **api**:
- `redirectUrlManagementApiBaseUrl` variable

File: src/Umbraco.Cms.Api.Management/Filters/ManagementApiJsonOutputFormatter.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 using System.Text.Json.Serialization;
 using Microsoft.AspNetCore.Mvc.Formatters;
 
-namespace Umbraco.Cms.ManagementApi.Filters;
+namespace Umbraco.Cms.Api.Management.Filters;
 
 public class ManagementApiJsonOutputFormatter : SystemTextJsonOutputFormatter
```
Comment (elit0451) [inline src/Umbraco.Cms.Api.Management/Filters/ManagementApiJsonOutputFormatter.cs:7]: Same here with the name - `ApiManagementJsonOutputFormatter`

**Comment by kjac** [comment]
@elit0451 I get where you're going, but I do actually think `ManagementApiXXX` is a more meaningful naming than `ApiManagementXXX`. At least for the time being. One could also argue that we don't need to prefix anything with `ManagementApi` since it's all inside a meaningful namespace.

If we do go ahead with renaming things we need to rename a few more classes:

image

Thoughts?

**Comment by elit0451** [comment]
All good, just wanted to check 😉 I am happy to merge the PR 💪

**Comment by kjac** [comment]
@elit0451 let's do it then 😆 you wanna do the honors?

**Comment by elit0451** [comment]
Yes please 💁‍♀️ 😁

**Pull request merged**